### DISCUSSION TITLE
please   add   the  function   of  add to    quick access

### DISCUSSION THREAD

**Discussion by Super1Windcloud** [spacedriveapp/spacedrive#2750] [Feature Request]
please   add   the  function   of  add to    quick access

**Comment by iLynxcat** [comment]
this is in our line of sight! will most likely not be added in 0.5.0, but we are targeting a subsequent release for it. (for other Spacedrive team members who might read this, this is [ENG-301 Composable Sidebar](https://linear.app/spacedriveapp/issue/ENG-301/composable-sidebar))
